src/Libraries/CoreNodeModels/Input/DateTime.cs

```diff
@@ -10,6 +10,7 @@ namespace CoreNodeModels.Input
     [NodeName("Date Time")]
     [NodeDescription("DateTimeDescription", typeof(Properties.Resources))]
     [NodeCategory(BuiltinNodeCategories.CORE_INPUT)]
+    [OutPortTypes("var")]
```

Comment:
Can we change this to 'dateTime'. I know that all the other 'dateTime' nodes have 'var' as output, but that doesn't really mean anything.